chain/network/benches/graph.rs

```diff
@@ -70,6 +67,6 @@ benchmark_group!(
 benchmark_main!(benches);
 
 // running 3 tests
-// test calculate_distance_10_10  ... bench:   1,503,222 ns/iter (+/- 126,811)
-// test calculate_distance_10_100 ... bench: 988,705,595 ns/iter (+/- 118,318,208)
-// test calculate_distance_3_3    ... bench:      18,928 ns/iter (+/- 3,174)
+// test calculate_distance_10_10  ... bench:      12,442 ns/iter (+/- 307)
```

Comment:
wow.. such a difference just due to the reference change? or we simply didn't update the test for a while

Reply:
We didn't update it for a while.  There were a lot of changes.